File: Practica4/Calcula_nota.c
#include <stdio.h>
void main () {
	int num;
	printf("Escribe la nota numérica: ");
	scanf("%d", &num);
	switch (num){
		case 5:
		case 6:
			printf("Aprobado\n");
			break;
		case 7:
		case 8:
			printf("Notable\n");
			break;
		case 9:
		case 10:
			printf("Sobresaliente\n");
			break;
		default:
			printf("Suspenso\n");
	}
}

File: Practica9/Calcula_nota.py
nota = int(input("Escriba una nota sobre 10:"))
if nota < 0 or nota > 10:
	print("El valor es incorrecto, introduzca un número del 0 al 10.")
print(calcula_nota(nota))
def calcula_nota(nota):
	if nota < 5 and nota >= 0:
		resultado = "Suspenso"
	elif nota == 5 or nota == 6:
		resultado = "Aprobado"
	elif nota == 7 or nota == 8:
		resultado = "Notable"
	elif nota == 9 or nota == 10:
		resultado = "Sobresaliente"
	else:
		resultado = "Nota inválida"
	return resultado
